migration/sql-files/068-limit-ooe-to-space.sql

```diff
@@ -0,0 +1,29 @@
+CREATE OR REPLACE FUNCTION limit_ooe_to_space() RETURNS void as $$
```

Comment:
@hectorj2f you can find migration test case here -> https://github.com/fabric8-services/fabric8-wit/pull/1478/files#diff-8300fcfb905de8543555d4fff40a7c2fR458